### PR TITLE
Clarify inheritance of app settings

### PR DESCRIPTION
**Pull request by crandmck**
Per #567

### CODE REVIEW

**Comment by crandmck** [comment]
@tunniclm Can you please review quickly?  I'll resolve the conflicts and merge if it looks OK.

File: _includes/api/en/4x/app-settings.md [hunk 1]
```diff
@@ -1,25 +1,41 @@
-If `name` is one of the application settings, it affects the behavior of the application. The following table lists application settings.
+The following table lists application settings.
+
+Note that sub-apps will:
+
+* Not inherit the value of settings that have a default value.  You must set the value in the sub-app.
+* Inherit the value of settings with no default value; these are explicitly noted in the table below.
+
+Exceptions: Sub-apps will inherit the value of `trust proxy` even though it does not have a default value (for backward-compatibility);
```
Comment (tunniclm) [inline _includes/api/en/4x/app-settings.md]: Wrong way round I think : Sub-apps will inherit the value of `trust proxy` even though it **has** a default value.

EDIT: You said it correctly further down. :)

**Comment by tunniclm** [comment]
Looks good except for the one comment I attached to the commit.

**Comment by crandmck** [comment]
Thanks for catching that @tunniclm!

**Pull request merged**